cli/cli_click.go

```diff
@@ -2,14 +2,31 @@ package cli
 
 import (
 	"bufio"
+	"context"
 	"fmt"
+	"net/url"
 	"os"
 
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
 	"github.com/urfave/cli/v2"
 )
 
 // Click is hidden cmd used to open some info when clicking on desktop icon
 func (c *cmd) Click(ctx *cli.Context) error {
+	if ctx.NArg() == 1 {
```

Comment:
when there is an error, shouldn't the window remain open and wait for key pressed? 
Otherwise the user would not know why it failed.

Reply:
Imo it should, but it mirrors the behavior for login, which doesn't display wait for user input. It would be best to just wait for key press in every case(success or failure), but I'm not sure why it wasn't implemented that way from the beginning.